apps/passport-server/src/routing/routes/telegramRoutes.ts

```diff
@@ -23,45 +27,55 @@ export function initTelegramRoutes(
    * an invite to the chat. In that case, we redirect the user back to
    * Telegram.
    */
-  app.get("/telegram/verify/:id", async (req: Request, res: Response) => {
-    try {
-      const { proof } = req.query;
-      const telegram_user_id = checkUrlParam(req, "id");
-      if (!proof || typeof proof !== "string") {
-        throw new Error("proof field needs to be a string and be non-empty");
-      }
+  app.get(
+    "/telegram/verify/:id/:username?",
```

Comment:
hmm it's a little annoying because the request is constructed and called with the `query` param in this helper https://github.com/proofcarryingdata/zupass/blob/525618c02ddd82b9d8f9c8b7c8bb19eef41f47ca/apps/passport-server/src/util/telegramHelpers.ts#L426-L433 so I'd need to thread it through https://github.com/proofcarryingdata/zupass/blob/525618c02ddd82b9d8f9c8b7c8bb19eef41f47ca/packages/passport-interface/src/PassportInterface.ts#L74 as an optional param. Not that bad, but it's working fine as is hehe